django/template/context.py

```diff
@@ -31,7 +31,9 @@ def __init__(self, dict_=None):
     def _reset_dicts(self, value=None):
         builtins = {"True": True, "False": False, "None": None}
         self.dicts = [builtins]
-        if value is not None:
+        if isinstance(value, BaseContext):
+            self.dicts.extend(value.dicts[1:])
```

Comment:
I don't think it **matters**, but the `push` method (further down) does much the same thing, but in a different _style_ which would (I believe) look like:
```python
self.dicts += value.dicts[1:]
```

Just pointing it out. It amounts to the same thing either way 🤷

Reply:
No problem, I made that update so changing it back 👍